vademecum_VT.md

```diff
@@ -56,7 +56,7 @@ Inoltre, A-Frame, ha diversi sottoprogetti come _A-Terrain_, per visualizzare da
 Rust è un linguaggio compilato con performance simili a quelle del C e garantisce, in aggiunta, la sicurezza del programma. È sviluppato da Mozilla ed è un componente fondamentale in Firefox Quantum.
```

Comment:
Cosa vuol dire sicurezza del programma?

Reply:
Perfetto

Reply:
Sicurezza è traduzione venuta male di linguaggio safe, che penso che sia fondamentale come concetto per rust. Non ho idea come si possa dire meglio in italiano

Reply:
io lo eviterei visto che poi dopo ci sono dei termini tecnici che spiegano meglio quella parte